captcha-validation-service/pom.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.mosip.esignet</groupId>
+        <artifactId>esignet-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+	<groupId>io.mosip.esignet.captcha</groupId>
+	<artifactId>captcha-validation-service</artifactId>
+
+	<name>captcha-validation-service</name>
+	<description>A pre-registration service to generate and validate captcha</description>
```

Comment:
Remove usage of pre-registration

Reply:
removed teh usage of preregistration from pom file